python/lsst/ip/isr/crosstalk.py

```diff
@@ -98,11 +111,23 @@ def __init__(self, detector=None, nAmp=0, **kwargs):
                                  dtype=int) if self.nAmp else None
         self.coeffValid = np.zeros(self.crosstalkShape,
                                    dtype=bool) if self.nAmp else None
+        # Quadratic terms, if any.
+        self.coeffsSqr = np.zeros(self.crosstalkShape) if self.nAmp else None
+        self.coeffErrSqr = np.zeros(self.crosstalkShape) if self.nAmp else None
+
+        # Gain ratios
+        self.ampGainRatios = np.zeros(self.crosstalkShape) if self.nAmp else None
+
+        # Units
+        self.crosstalkRatiosUnits = 'electrons' if self.nAmp else None
```

Comment:
I was just because the coefficients provided by Adam in this ticket are in those units. I'm happy to set the default to ADU, is that what we want? @erykoff

Reply:
So the proper unit name is `electron` not `electrons`.  (This should be fixed elsewhere as well).  Adam's crosstalk coefficients were computed after applying gain so they are in units of electrons and not ADU.  However, I don't think this should necessarily be the default if unspecified (though Chris should answer if previous crosstalk matrices were generated before or after gain correction).

Reply:
(The proper unit name for ADU is `adu`)

Reply:
@czwa, should we set the default to ADU? (`adu` in the code).

Reply:
I chaged the defalt to `adu`. Does "proper" mean "as in astropy", in this context?

Reply:
All existing CT has been measured pre-gain application, without flats, so I think they should be in ADU.